standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java

```diff
@@ -2276,6 +2277,9 @@ private void create_table_core(final RawStore ms, final CreateTableRequest req)
           + " already exists");
     }
 
+    tbl.setDbName(normalizeIdentifier(tbl.getDbName()));
```

Comment:
The lineage info will feed into downstream project, so I'm a little nervous when the info changes.
I think we can make changes here like:
```java
    Table tbl = new Table(req.getTable());
    tbl.setDbName(normalizeIdentifier(tbl.getDbName()));
    tbl.setTableName(normalizeIdentifier(tbl.getTableName()));

Reply:
It tried to implement your suggestion but 29 tests are failing because the tests are doing assert on the Table to construct a table path or table SD or something that is expected to be modified in HMS. So I think this change is not necessary.
Having changes in qfile files that change lineage info (which is anyway expected) is far better than changing 29 tests to do something else. Thanks.